database/index.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require ('mongoose');
```

Comment:
airbnb style guide recommends a space between require statements and other variable initializations
```suggestion
 const mongoose = require ('mongoose');
 
```